docs/pipeline/sources/update.mdx

```diff
@@ -12,12 +12,23 @@ It is used to identify the service that emitted a span, metric or log record.
 
 Odigos will automatically populate this attribute based on the name of the workload. If there are multiple containers in the workload, the `service.name` will be set to the container name.
 
-If odigos fails to set a `service.name`, or if you want to override the default value, you can set it manually for a specific source.
-
+If odigos fails to set a `service.name`, or if you want to override the default value, you can also set the `service.name` attribute by adding the `otelServiceName` attribute to an exiting Source, or by creating a `Source` with that attribute:
```

Comment:
nit:

The "fails to set a `service.name`" is here because of the device id uuid which sometimes leaked into the telemetry. I think it is no longer relevant (mechanism replaced everywhere), so we can remove these negative words and just suggest the user to choose his own value.